app/views/characters/search.haml

```diff
@@ -14,15 +14,15 @@
       %tr
         %td.width-300.padding-10.vtop.dark
           = form_tag search_characters_path, :method => :get do
-            %table#signup
+            %table.search-form
```

Comment:
I think this was for styling reasons, did formatting change too?

Reply:
The only style that applies to `#signup` is `#signup .sub { width: 90px; }` – I'll look if there are any `.sub`s but I didn't notice a change, at least.

Reply:
Looking at the differences between master and this branch doesn't show any visible change (except for a darkened input box). It looks like `#signup` wasn't necessary.